invenio_rdm_pure/ext.py

```diff
@@ -35,19 +35,17 @@ def init_config(self, app):
             )
         for k in dir(config):
             if (
-                k.startswith("INVENIO_RDM_PURE_")
-                or k.startswith("INVENIO_PURE")
-                or k.startswith("PURE")
+                k.startswith("INVENIO_RDM_PURE_") or k.startswith("INVENIO_PURE") or k.startswith("PURE")
             ):
                 app.config.setdefault(k, getattr(config, k))
         if "SQLALCHEMY_DATABASE_URI" in app.config:
             database_uri = app.config.get("SQLALCHEMY_DATABASE_URI").split("//")[1]
```

Comment:
Why is it okay to access the second element here, but not in the rest of this part?

Reply:
@bengaal -  I think there is no expected value as ```SQLALCHEMY_DATABASE_URI=postgresql+psycopg2://username:inveniordm@127.0.0.1:5432/invenio-app-rdm```. And the fact that it does not return error on the first or second element - only logical explanation would be that there is a default value assigned to the variable and that value does not have other indexes as we expect. And as far as I remember it should use a default value as such ```SQLALCHEMY_DATABASE_URI=sqlite://```.

Reply:
The problem isn't present anymore with the original version. So I will reset these array accesses.